source/features/sticky-csv-header.css

```diff
@@ -13,8 +13,8 @@
 
 .blob-wrapper.type-csv,
 .Box:has(.csv-data) {
-	border-bottom-right-radius: 0;
-	border-bottom-left-radius: 0;
+	border-bottom-right-radius: none;
+	border-bottom-left-radius: none;
```

Comment:
This is not allowed, see https://developer.mozilla.org/en-US/docs/Web/CSS/border-radius

Also see https://github.com/xojs/stylelint-config-xo/pull/17, probably you have an old version locally.

Reply:
That's explain why CI wasn't failing. 👌 

Feel free to "restore the file" once you accept the membership 😉

Reply:
Both done. Thanks for inviting me :)